lib/docs/filters/core/entries.rb

```diff
@@ -56,8 +56,16 @@ def build_entries(entries)
 
     def build_entry(name, frag = nil, type = nil)
       type ||= self.type
-      path = frag ? (frag.include?('#') ? frag : "#{self.path}##{frag}") : self.path
-      Entry.new(name, path, type)
+      if frag.present? && frag.include?('#')
+        path = frag
+        # TODO: What should `url` get changed to?
```

Comment:
I’m trying to figure out what this should do. I can’t find any examples of this in other documentation, so I don’t know what behavior it has.